core-rust/core-api-server/src/core_api/conversions/addressing.rs

```diff
@@ -376,17 +368,19 @@ fn to_mapped_substate_id(substate_id: SubstateId) -> Result<MappedSubstateId, Ma
                         SubstateKeyType::AccessRulesChain,
                     ),
                 },
+                SubstateOffset::PackageAccessRules => (
+                    SubstateType::FunctionAccessRules,
+                    SubstateKeyType::FunctionAccessRules,
+                ),
```

Comment:
This mapping was missing, according to `TransactionStreamTest.test_core_api_can_submit_and_commit_transaction()`.
I made a guess based on: https://github.com/radixdlt/radixdlt-scrypto/blob/840505d6b0d2d4fe4127c0dbaf12243467155fb9/radix-engine/src/system/node_substates.rs#L259
Please verify ❓